simtools/corsika/corsika_output_visualize.py

```diff
@@ -29,65 +30,64 @@ def _kernel_plot_2D_photons(corsika_output_instance, property_name, log_z=False)
     -------
     list
         List of figures for the given telescopes.
+    list
+        List of the figure names.
 
     Raises
     ------
     ValueError
-        if `name` is not allowed.
-
-    """
-    x_label = {
-        "counts": "x (m)",
-        "density": "x (m)",
-        "direction": "cos(x)",
-        "time_altitude": "Time since 1st interaction (ns)",
-        "num_photons_per_telescope": "Event number",
-    }
-    y_label = {
-        "counts": "y (m)",
-        "density": "y (m)",
-        "direction": "cos(y)",
-        "time_altitude": "Altitude of emission (km)",
-        "num_photons_per_telescope": "Telescope index",
-    }
-    if property_name not in x_label:
-        msg = f"property_name must be one of {list(x_label.keys())}"
+        if `property` is not allowed.
+    """
+    if property_name not in corsika_output_instance._dict_2D_distributions:
+        msg = f"This property does not exist. The valid entries are " \
```

Comment:
Please do not use `\` to break strings, we decided a while back only to use `()`